cylc/flow/scripts/cylc.py

```diff
@@ -136,8 +136,8 @@ def get_version(long=False):
         'cylc get-config has been replaced by cylc config',
     'get-site-config':
         'cylc get-site-config has been replaced by cylc config',
-    'get-suite-config':
-        'cylc get-suite-config has been replaced by cylc config',
+    'get-workflow-config':
+        'cylc get-workflow-config has been replaced by cylc config',
```

Comment:
```suggestion
    'get-suite-config':
        'cylc get-suite-config has been replaced by cylc config',
```

Also I think an alias or dead end for `suite-state` (now `workflow-state`) should be added

Reply:
Done.